### ISSUE TITLE
Invalid value: Not in inclusive range

### ISSUE THREAD

**Issue by martesabt** [HeavenOSK/flutter_swipable_stack#34]
Hi,

I recently noticed that the package has a new version (**0.8.0**) so I decided to update from version **0.7.1** to **0.8.0**.

Now I got this error when I swipe first card:

```
The following RangeError was thrown building LayoutBuilder:
RangeError (index): Invalid value: Not in inclusive range 0..2: 3

The relevant error-causing widget was: 
  SwipableStack-[LabeledGlobalKey<_SwipableStackState>#fa547] file:///[MY_PROJECT]/lib/src/ui/screens/home/home.dart:74:12
When the exception was thrown, this was the stack: 
#0      List.[] (dart:core-patch/growable_array.dart:254:60)
#1      _HomeScreenState._swipeableContent.<anonymous closure> (package:[MY_PROJECT]/src/ui/screens/home/home.dart:81:34)
#2      _SwipableStackState._buildCards.<anonymous closure> (package:swipable_stack/src/swipable_stack.dart:395:37)
#3      new _GrowableList.generate (dart:core-patch/growable_array.dart:133:28)
#4      _SwipableStackState._buildCards (package:swipable_stack/src/swipable_stack.dart:391:19)
```

I can confirm that with version 0.7.1 the code and package works perfectly :)

PS: I can bring more code or info if needed.

**Comment by HeavenOSK** [comment]
@martesabt 
Thanks for creating issue. I'll investigate about it 🙏

**Comment by HeavenOSK** [comment]
I've reproduced this problem. Thanks you so much 🙏 
  
iPhone Simulator |
:-: | 
<img src='https://user-images.githubusercontent.com/19836917/147021120-cd196f46-9181-46fb-93a9-0824e0d292e3.png' width=180/> |

Please wait a little bit, I'll fix it on `v0.8.1` in 24 hours.

**Comment by martesabt** [comment]
@HeavenOSK Thanks! I will wait to test the next version :)

**Comment by HeavenOSK** [comment]
@martesabt 
I published [swipable_stack 0.8.1](https://pub.dev/packages/swipable_stack/versions/0.8.1) to apply bugfix for this issue.
Thanks for the bug report 🙏

**Comment by martesabt** [comment]
@HeavenOSK Thank you for fast fixing!

It works perfect with the new version!

Thank you again!!!
